packages/proxy/src/proxy.ts

```diff
@@ -2008,6 +2008,36 @@ async function fetchOpenAI(
     }
   }
 
+  if (secret.type === "mistral" && stream && bodyData?.stream) {
```

Comment:
i don't think we actually want to discard them though. Users want to see the reasoning in the UI, just like you can with Anthropic models. Instead, i think we want to append it to the content block (after parsing it to just get the text part of it), like we do for anthropic here: https://github.com/braintrustdata/braintrust/blob/enable-advanced-analytics/proxy/packages/proxy/src/providers/anthropic.ts#L404

Reply:
makes sense, i've addressed it.